test/scripts/generate-ostree-build-config

```diff
@@ -33,6 +34,15 @@ def read_config_map():
         return json.load(config_map_file)
 
 
+def matches(a: str, items: list[str]) -> bool:
+    matched = False
+    for item in items:
+        if fnmatch.fnmatch(a, item):
+            matched = True
```

Comment:
Just return True here, and False at the end. Then you can drop `matched` variable.
Or you could simplify it to:
`return any(fnmatch.fnmatch(a, i) for i in items)`